server/client.go

```diff
@@ -1917,9 +1917,9 @@ func (c *client) traceMsg(msg []byte) {
 
 	if maxTrace > 0 && l > maxTrace {
 		tm := fmt.Sprintf("%q", msg[:maxTrace])
-		c.Tracef("<<- MSG_PAYLOAD: [\"%s...\"]", tm[1:maxTrace+1])
+		c.Tracef("<<- MSG_PAYLOAD: [\"%s...\"]", tm[1:len(tm)-1])
```

Comment:
Changed this since when converting to the quoted form, `\r\n` is escaped and thus four characters.